docs/intro/glossary.md

```diff
@@ -0,0 +1,237 @@
+# Glossary
+
+This glossary provides definitions for common terminology used in the SDK,
+focusing on microgrid components, metrics, measurements, and power-related
+terms.
+
+## Common Acronyms
+
+### AC
+
+Alternating current. See the [Wikipedia article](https://en.wikipedia.org/wiki/Alternating_current) for more details.
+
+### BMS
+
+Battery management system. See the [Wikipedia article](https://en.wikipedia.org/wiki/Battery_management_system) for more details.
+
+### CHP
+
+Combined heat and power. See the [Wikipedia article](https://en.wikipedia.org/wiki/Combined_heat_and_power) for more details.
+
+### DC
+
+Direct current. See the [Wikipedia article](https://en.wikipedia.org/wiki/Direct_current) for more details.
+
+### EV
+
+Electric vehicle. See the [Wikipedia article](https://en.wikipedia.org/wiki/Electric_vehicle) for more details.
+
+### PSC
+
+Passive sign convention. See the [Wikipedia article](https://en.wikipedia.org/wiki/Passive_sign_convention) for more details.
+
+### PV
+
+Photovoltaic. See the [Wikipedia article](https://en.wikipedia.org/wiki/Photovoltaics) for more details.
+
+In the SDK it is normally used as a synonym for [solar panel](#solar-panel).
+
+### SoC
+
+[State of charge](#state-of-charge). See the [Wikipedia article](https://en.wikipedia.org/wiki/State_of_charge) for more details.
+
+### SoP
+
+[State of power](#state-of-power).
+
+## Microgrid
+
+A local electrical grid that connects a set of different [types of components](#component-category) together. It can be connected to the main grid, or be completely isolated.
+
+### Component Category
+
+The different types of [components](#component) a [microgrid](#microgrid) can have, such as an inverter, battery, meter, etc.
+
+Devices of the same category have the same characteristics (for example offer the same set of [metrics](#metric)).
+
+#### Battery
+
+A storage system for electrical energy.
+
+#### CHP Plant
+
+A generator that produces combined heat and power ([CHP](#chp)). Usually powered via combustion of some form of fuel.
+
+#### Converter
+
+A [DC](#dc)-DC converter.
```

Comment:
hmm DC to DC converter? :thinking: 
To avoid confusion I'd suggest to write it as follows:
```suggestion
A [DC](#dc) converter.
```

Reply:
I think DC to DC (or DC-DC) is what's correct, @tiyash-basu-frequenz ?

Reply:
I went with wikipedia: https://en.wikipedia.org/wiki/DC-to-DC_converter

Added a definition for `DC-to-DC converter` and said that `Converter` usually refers to that.

Reply:
DC-DC converter is actually the term used in the industry.

Reply:
oh I didn't know. I admit it sounded confusing to me.